essays.html

```diff
@@ -33,6 +33,11 @@ <h1><a href='index.html'>CSC630: The Open Source Movement</a> // Assignments <sp
           A collection of essays written by students in this class. To be updated once said essays have been written.
         </p>
       </article>
+        <p>
+            <a href = "https://docs.google.com/document/d/1o7zNZMET9MhhfSaC6dp1AKPBOgGrBZXy5xgHYp8r5X8/edit?usp=sharing"> Open Source and Diversity - Jocelyn Shen </a>
+        </p>
+      <article>
```

Comment:
Perhaps this tag should go before the opening `<p>` tag? Right now it's just an empty `<article></article>`